library/control/test/ProductFeatures_test.rb

```diff
@@ -158,4 +158,95 @@
       end
     end
   end
+
+  describe "#GetFeature" do
+    let(:scr_root_dir) { File.join(File.dirname(__FILE__), "data") }
+    let(:normal_stage) { false }
+    let(:firstboot_stage) { false }
+
+    before do
+      allow(Yast::Stage).to receive(:normal).and_return(normal_stage)
+      allow(Yast::Stage).to receive(:firstboot).and_return(firstboot_stage)
+    end
+
+    around do |example|
+      change_scr_root(scr_root_dir, &example)
+    end
+
+    context "when features are not initialized yet" do
+      it "initializes them" do
+        expect(subject.GetFeature("globals", "base_product_license_directory")).to_not be_nil
+      end
+    end
+
+    context "when features are already initialized" do
+      before do
+        subject.InitFeatures(true)
+      end
+
+      it "does not force to initialize them again" do
+        expect(subject).to receive(:InitFeatures).with(false)
+
+        subject.GetFeature("globals", "base_product_license_directory")
+      end
+
+      context "in normal stage" do
```

Comment:
Does it apply only when _features are already initialized_? Because as you stated in the outer context, they will be initialized anyway when calling `GetFeature`. I would move these examples to the outer context.